OTEP.md

```diff
@@ -17,26 +19,38 @@ The environment variable interface suffers from the following identified limitat
 * **Limited validation**. Validation can only be performed by the receiver, there is no meta-configuration language to validate input.
 * **Difficult to extend**. It’s difficult to anticipate the requirements of configuring custom extension components (processors, exporters, samplers, etc), and likely not practical to represent them in a flat structure. As a result, the environment variable interface is limited to components explicitly mentioned in the specification.
 
-## Explanation (TBD)
+## Explanation
 
-Using a configuration model or configuration file, users could configure all options currently available via environment variables.
+Using a configuration model or configuration file, users can configure all options currently available via environment variables.
 
 ### Goals
 
-* The configuration file must be language implementation agnostic. It must not contain structure or statements that only can be interpreted in a subset of languages. This does not preclude the possibility that the configuration file can have specific extensions included for a subset of languages, but it does mean that the standard format of the file must be interpretable by all implementation languages.
+* The configuration must be language implementation agnostic. It must not contain structure or statements that only can be interpreted in a subset of languages. This does not preclude the possibility that the configuration can have specific extensions included for a subset of languages, but it does mean that the standard format must be interpretable by all implementation languages.
 * Broadly supported format. Ideally, the information encoded in the file can be decoded using native tools for all OpenTelemetry implementation languages. However, it must be possible for languages that do not natively support an encoding format to write their own parsers. This means that the file encoding format must be specified in a language agnostic form.
-* The file format must support structured data. At the minimum arrays and associative arrays.
-* The file format must support at least null, string, double precision floating point (IEEE 754-1985), or signed 64 bit integer value types.
-* Extensible. Custom span processors, exporters, samplers, or other user defined code can be configured using this format.
+* The configuration format must support structured data. At the minimum arrays and associative arrays.
+* The format must support at least null, string, double precision floating point (IEEE 754-1985), or signed 64 bit integer value types.
+* Custom span processors, exporters, samplers, or other user defined code can be configured using this format.
 * Configure SDK, but also configure instrumentation.
-* Versioning: needs to be able to version stability while evolving
-* (stretch) The file format can be validated client side.
+* It needs to be able to version stability while evolving
```

Comment:
I figured default values would continue to work the way they currently do in the various languages. Do you think there would be a need to add any kind of definition or leave it up to implementations?

Reply:
Well the defaults for the languages are defined as part of the environment variable schema. I.e. we know that the default interval for the batch span processor is 5000ms because its defined in the [OTLP_BSP_SCHEDULE_DELAY](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md#batch-span-processor) spec.

Maybe is enough to just say that the defaults from the environment variable schema apply here as well?

Although, someday file configuration surface area will exceed the environment variable configuration surface area. At that point, we'll need to define them here. A quick search revealed that JSON schema has a [default](https://json-schema.org/understanding-json-schema/reference/generic.html#annotations) keyword which has no impact on validation, but which is meant to convey semantics to users in case a property value is missing.